src/mlpack/methods/ann/gan/gan_impl.hpp

```diff
@@ -217,14 +217,14 @@ template<
   typename Noise,
   typename PolicyType
 >
-template<typename OptimizerType>
-double GAN<Model, InitializationRuleType, Noise, PolicyType>::Train(
-    arma::mat trainData,
-    OptimizerType& Optimizer)
+template<typename OptimizerType, typename... CallbackTypes>
+double GAN<Model, InitializationRuleType, Noise,
+           PolicyType>::Train(OptimizerType& Optimizer,
+           CallbackTypes&&... callbacks)
 {
-  ResetData(std::move(trainData));
-
-  return Optimizer.Optimize(*this, parameter);
+  if (!reset)
+    Reset();
```

Comment:
Why this change?

Reply:
Minor mistake on my side. I fixed it now. But it looks like the travis build timed out again.